WORKSPACE

```diff
@@ -268,6 +268,24 @@ rbe_autoconfig(
     use_checked_in_confs = "False",
 )
 
+rbe_autoconfig(
+    name = "rbe_autoconf_generate_windows",
+    bazel_version = "3.1.0",
+    create_testdata = True,
+    use_checked_in_confs = "False",
+    digest = "sha256:4fa38fa8c1070850b098d8c03334a86774fffa87004a0ce601d87a883745f9f0",
+    registry = "gcr.io",
+    repository = "envoy-ci/envoy-build-windows",
+    exec_compatible_with = [
```

Comment:
Is it acceptable to set the default values for `exec_compatible_with` and `target_compatible_with` (or any other arguments that might generate linux specific config) to Windows values so users don't have to specify these? Or should we keep the defaults as the Linux defaults and make users override them?